### PR TITLE
Hide cluster name in tooltip if cells are coloured

### PR DESCRIPTION
**Pull request by lingyun1010**
… by metadata

### CODE REVIEW

**Review by monicajianu — COMMENTED** [review]
While technically correct, I think you could achieve a more elegant solution using the `selectedColourByCategory` prop available to you in the `TsnePlotView` component. Checking contents of strings is never a good idea, especially since that string is set in the backend.

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
Please, add a test for this case.

File: src/ClusterTSnePlot.js [hunk 1]
```diff
@@ -23,6 +23,12 @@ const _colourizeClusters = (highlightSeries) =>
     }
   })
 
+const tooltip_header = (clusterType, series, point) => {
```
Comment (alfonsomunozpomer) [inline src/ClusterTSnePlot.js]: Please, use `camelCase` for the function name.

**Review by alfonsomunozpomer — APPROVED** [review]
Good! There’s some risk that untested code becomes code rot. Remember to add tests for any new features you add, especially in packages where we already have tests. There are others which are beyond any hope and we will reimplmenet them in time.

Remember to change the function name to keep the linter happy.

**Pull request merged**